README.md

```diff
@@ -28,14 +28,14 @@ A `docker-compose.yml` looks like this:
       web:
         build: .
         ports:
-         - "5000:5000"
```

Comment:
The quotes are consistent with [our advice regarding port mappings](https://docs.docker.com/compose/compose-file/#short-syntax-1). 

> Note: When mapping ports in the `HOST:CONTAINER` format, you may experience erroneous results when using a container port lower than 60, because YAML parses numbers in the format `xx:yy` as a base-60 value. For this reason, we recommend always explicitly specifying your port mappings as strings.

In this instance, even though the two are equivalent, I think we should "practice what we preach" and use the quoted version.

Reply:
That's hard to argue against, okay :+1: